src/index.html

```diff
@@ -1,13 +1,290 @@
+<!-- #region head-->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>Nothing</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.cdnfonts.com/css/space-grotesk" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Space+Mono&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content ">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo">
+              <img src="images/logo.svg" alt="Nothing logo">
+            </a>
+
+            <div class="top-bar__icons">
+              <a href="tel:+1 234 5555-55-55" class="icon icon--phone"></a>
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+
+          <div class="header__bottom">
+            <h1 class="header__title">Bring joy back to the everyday</h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo">
+            <img src="images/logo.svg" alt="Nothing logo">
+          </a>
+          <a href="#" class="icon icon--close"></a>
+        </div>
+
+        <div class="menu__bottom ">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#products">Products</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#store">Store</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#login">Login</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#cart">Cart(0)</a>
+              </li>
+            </ul>
+          </nav>
+
+          <a href="tel:+1 234 5555-55-55" class="menu__phone-number">
```

Comment:
tel:+1 234 5555-55-55 for attribute href on element a: space is not allowed accordingly to the standard